### ISSUE TITLE
Multi-class multi-object Support?

### ISSUE THREAD

**Issue by DataXujing** [Zhongdao/Towards-Realtime-MOT#138]
In the `Dataset Zoo` I see 

> The field [class] should be 0. Only single-class multi-object tracking is supported in this version.

means only support single-class, **Can I tracking person and car in the meantime ？**

**Comment by Zhongdao** [comment]
Thanks for your interest in our work! The current version of JDE only supports single-class MOT. For multi-class MOT, we are working on an extended version of JDE, please stay tuned.

**Comment by abhaygargab** [comment]
Hello @Zhongdao ,
Has the work on MCMOT version completed? 

ThankYou

**Comment by JAYCHOU2020** [comment]
> Thanks for your interest in our work! The current version of JDE only supports single-class MOT. For multi-class MOT, we are working on an extended version of JDE, please stay tuned.

Has the work on MCMOT version completed?

ThankYou

**Comment by Zhongdao** [comment]
Hi, @JAYCHOU2020,
Our new project [UniTrack](https://github.com/Zhongdao/UniTrack) supports multi-class multi-object tracking now. But It is not a direct extension of JDE actually. Please check out our paper and the YOLOX+UniTrack demo for more details.

**Comment by JAYCHOU2020** [comment]
> Hi, @JAYCHOU2020,
> Our new project [UniTrack](https://github.com/Zhongdao/UniTrack) supports multi-class multi-object tracking now. But It is not a direct extension of JDE actually. Please check out our paper and the YOLOX+UniTrack demo for more details.



> Hi, @JAYCHOU2020,
> Our new project [UniTrack](https://github.com/Zhongdao/UniTrack) supports multi-class multi-object tracking now. But It is not a direct extension of JDE actually. Please check out our paper and the YOLOX+UniTrack demo for more details.

So,is this code Towards-Realtime-MOT code can be modify to MCMOT?

**Comment by Zhongdao** [comment]
> > Hi, @JAYCHOU2020,
> > Our new project [UniTrack](https://github.com/Zhongdao/UniTrack) supports multi-class multi-object tracking now. But It is not a direct extension of JDE actually. Please check out our paper and the YOLOX+UniTrack demo for more details.
> 
> > Hi, @JAYCHOU2020,
> > Our new project [UniTrack](https://github.com/Zhongdao/UniTrack) supports multi-class multi-object tracking now. But It is not a direct extension of JDE actually. Please check out our paper and the YOLOX+UniTrack demo for more details.
> 
> So,is this code Towards-Realtime-MOT code can be modify to MCMOT?

Of course, as long as you have adequate training data. See [this repo](https://github.com/CaptainEven/MCMOT).
